src/Config.js

```diff
@@ -82,6 +93,8 @@ class ConfigOptions {
     votingDelayMin() { return this._votingDelayMin; }
     stagingChecks() { return this._stagingChecks; }
     loggerParams() { return this._loggerParams; }
+    // returns a Map of (login,id) pairs
+    coreDeveloperIds() { return this._coreDeveloperIds; }
```

Comment:
AFAICT, nobody really wants this method right now. What the callers want is something like this:

```JS
// whether the given GitHub user is a core developer
coreDeveloper(role, userLogin, userId)
{
    const coreId = Config.coreDeveloperIds().get(userLogin);
    if (!coreId)
        return false;
    this._log(`found core ${role}: ${userLogin}={userId}`);
    // die if we are misconfigured and/or the userLogin has moved to another user
    assert.strictEqual(coreId, userId);
    return true;
}
```

Reply:
Done.